go.mod

```diff
@@ -11,7 +11,8 @@ require (
 	github.com/go-logr/logr v1.2.2
 	github.com/go-logr/zapr v1.2.2
 	github.com/golang/mock v1.6.0
-	github.com/google/go-cmp v0.5.6
+	github.com/golangci/golangci-lint v1.45.2
```

Comment:
hmm....still confused why this is needed here.

Reply:
nope they're here too:
```
[10:14:49]-[~/go/src/github.com/tinkerbell/boots]─[manny@zennix]> 
cat tools.go 
───────┬──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
       │ File: tools.go
       │ Size: 228 B
───────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   1   │ //go:build tools
   2   │ // +build tools
   3   │ 
   4   │ package tools
   5   │ 
   6   │ import (
   7   │     _ "github.com/golang/mock/mockgen"
   8   │     _ "github.com/golangci/golangci-lint/cmd/golangci-lint"
   9   │     _ "golang.org/x/tools/cmd/goimports"
  10   │     _ "golang.org/x/tools/cmd/stringer"
  11   │ )
───────┴──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
                                                                                                                                                                      
[10:14:51]-[~/go/src/github.com/tinkerbell/boots]─[manny@zennix]> 
grep -e golang/mock -e golang.org/x/tools -e golangci-lint go.mod 
go.mod:    github.com/golang/mock v1.6.0
go.mod:    github.com/golangci/golangci-lint v1.45.2
go.mod:    golang.org/x/tools v0.1.10
```